include/wx/webview.h

```diff
@@ -78,6 +78,20 @@ enum wxWebViewFindFlags
     wxWEBVIEW_FIND_DEFAULT =          0
 };
 
+enum wxWebViewIEEmulationLevel
```

Comment:
Currently this is used as return value or parameter type of `{Get,Set}EmulationLevel()` in this header, so it can't be done, but I'm not sure at all why do we need these emulation level functions here to be honest.